CompanyAddress.md

```diff
@@ -7,124 +7,73 @@ _______
-| street_address | The official street line of an address relative to the area, as specified by the `locality` or `postal_code` properties. **This property may be absent**. |
-| sublocality | The neighborhood, borough, township, etc. for the address. Sublocalities are typically more specific than cities and may be returned even if the locality field is empty. **This property may be absent**. |
-| landmark | The full name of the landmark (such as a military base or island) returned by the service. **This property may be absent**. |
-| place_id | Google's unique identifier string corresponding to the returned location. |
-| viewport | A bounding box describing a rectangle that encloses the location. The viewport field contains two coordinate objects -- **northeast** and **southwest** -- each with **lat** and **lng** decimal values. It also contains a **suggested_radius** decimal field that contains the distance in miles from the center of the viewport to a corner. **This property and its elements are always present and non-empty.** |
-| metropolitan_statistical_area | The metropolitan statistical area containing the location. An object containing two fields, the title of the area and an integer code for the area. Metropolitan statistical areas are specific to the US. **This property may be absent**. |
-| designated_market_areas | An object array of the designated market areas containing the location. Designated market areas may overlap and thus a location may be in more than one designated market area. Designated market areas are specific to the continental US. See below for details on the structure of these objects. **This property may be absent and the array may be empty**. |
-| partial_match | A boolean value indicating that the geocoder did not return an exact match. **This property will always be false for Company Address searches**.|
-
-&nbsp;
-
-Each element of the returned **admin_areas** array will be formatted as follows:
-
-|   |   |
-|---|---|
-| short_name | The abbreviated form of the administrative area's name. This will vary by country and administrative division level. In the US, states (admin_area level 1) will use the state abbreviation (e.g. "GA"), and counties will use the full county name (e.g. "Gwinnett County"). **This property is always present and non-empty.** |
-| long_name | The unabbreviated form of the administrative area's name. **This property is always present and non-empty.** |
-| level | An integer between 1 and 5 inclusive specifying the hierarchal level at which this administrative area resides. In the US, states will be returned with a level of 1 and counties with a level of 2. **This property is always present and non-empty.** |
+## Request Structure
+Requests consist of:
 
-&nbsp;
+| Field Name        | Type   | Required |
+|-------------------|--------|----------|
+|`company_id`      | string | True      |
+|`country` | string | False       |
+|`state` | string | False       |
```

Comment:
Is this the full name or a 2 letter ISO identifier? Same with the country.

Reply:
I think from the sample request this is conveyed.